docs/_documentations/troubleshooting.md

```diff
@@ -461,6 +461,23 @@ Codewind displays an error. In VS Code, the error appears in the Codewind log:
 ***
 # OpenShift Do (odo) with Codewind
 
+For general information about the OpenShift Do (odo) extension on Codewind, see the [README](https://github.com/eclipse/codewind-odo-extension) file in the `codewind-odo-extension` repository.
+
+<!--
+Action/Topic: OpenShift Do (odo) with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/1514
+-->
+## Fail to build odo project due to existing image stream
+If you build odo project with the following type of error messages in the build log, this issue is due to the image stream you create already exists in your cluster.
+```
+Failed to create component with name cw-perlodoodo. Please use odo config view to view settings used to create component. Error: imagestreams.image.openshift.io "cw-perlodoodo-app" already exists unable to create ImageStream for cw-perlodoodo-app
+```
+
+**Workaround**
```

Comment:
Please add the `:` inside the asterisks.

Reply:
@jingfuwang 
Thanks for providing these instructions to the user.